unicodetools/src/main/java/org/unicode/text/UCD/UCD_Types.java

```diff
@@ -27,10 +27,16 @@ public interface UCD_Types {
     // Unicode 12:
     // 17000;<Tangut Ideograph, First>;Lo;0;L;;;;;N;;;;;
     // 187F7;<Tangut Ideograph, Last>;Lo;0;L;;;;;N;;;;;
+    // Unicode [..]:
```

Comment:
optional:
Instead of "REMOVE BEFORE FLIGHT" and "[..]", how about "TODO"?

Reply:
We merge TODOs sometimes (often even), I am going for something more like Google’s DO NOT SUBMIT.